locksmith/src/app.js

```diff
@@ -16,5 +17,6 @@ app.post(/^\/lock$/i, signatureValidationMiddleware)
 app.use('/', router)
 app.use('/', transactionRouter)
 app.use('/', lockRouter)
+app.use('/block', blockRouter)
```

Comment:
Is this route too specific? I.e., will we wish in the future that we reserved `/block` for something other than just the timestamp?

Reply:
The general thinking right now is that we will keep the same structure and expand the output to support more of our block centric/related use cases